.gitignore

```diff
@@ -0,0 +1 @@
+front
```

Comment:
why ignoring the front directory?

Reply:
its only for the PR, to push back-end dir to dev instead of pushing front again

Reply:
to not push front again, just don't push it: don't add it, don't commit it, don't push it